### PR TITLE
Add article 'Les femmes dans l'informatique'

### PR DESCRIPTION
**Pull request by MarieMinasyan**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
9 words might be misspelled, please check them: connait, développeuses, equite, formatage, Gehannin, lang, login, marianne, marishka

File: _posts/fr/2021-03-08-les-femmes-dans-linformatique.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+lang: fr
+layout: post
+title: 
+excerpt: Aujourd'hui, c'est la journée internationale des droits de la femme, et nous avons voulu en profiter pour vous parler d'un sujet qui nous tient à cœur : les femmes dans l'informatique. Le but de cet article est de vous raconter notre expérience pour vous donner une meilleure vision de pourquoi on n'est pas si nombreuses, pourquoi certaines abandonnent, et, surtout, comment faire pour que ça change.
```
Comment (VEBERArnaud) [inline _posts/fr/2021-03-08-les-femmes-dans-linformatique.md:5]:
```suggestion
excerpt: >
    Aujourd'hui, c'est la journée internationale des droits de la femme, et nous avons voulu en profiter pour vous parler d'un sujet qui nous tient à cœur : les femmes dans l'informatique. Le but de cet article est de vous raconter notre expérience pour vous donner une meilleure vision de pourquoi on n'est pas si nombreuses, pourquoi certaines abandonnent, et, surtout, comment faire pour que ça change.
```

File: _posts/fr/2021-03-08-les-femmes-dans-linformatique.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+> *En tant que femme, avoir choisi un métier masculin... c'est parce que tu aimes les hommes ?*
+ 
+2. Elle donne une formation. L'un des participants décroche son téléphone dans la salle et interrompt la formation 8 fois en une demi-journée. Elle attend que tout le monde parte manger pour lui demander de sortir de la salle s'il a un appel à prendre :
+> *T'es une femme et tu es plus jeune que moi, tu n'as pas à me dire quoi faire.*
+
+3. Elle fait passer un entretien technique à un candidat :
+> *T'es une femme, je ne sais pas si tu connais les principes SOLID.*
+
+4. Elle discute avec le collègue A de la meilleure solution technique à un problème. Le collègue B, qui ne connait ni le language de programmation ni le framework en question, lui propose une solution (inadaptée), elle le remercie et dit qu'ils gardent sa proposition en tête, mais peuvent peut-être trouver mieux (*et non, il ne rigolait pas*):
+> *Non mais, moi je suis un homme, toi t'es une femme, tu te fais et tu fais ce que je te dis.*
```
Comment (VEBERArnaud) [inline _posts/fr/2021-03-08-les-femmes-dans-linformatique.md:46]:
```suggestion
> *Non mais, moi je suis un homme, toi t'es une femme, tu te tais et tu fais ce que je te dis.*
```

**Comment by ElevenWilson** [comment]
9 words might be misspelled, please check them: connait, développeuses, equite, formatage, Gehannin, lang, login, marianne, marishka

**Comment by ElevenWilson** [comment]
9 words might be misspelled, please check them: connait, développeuses, equite, formatage, Gehannin, lang, login, marianne, marishka

**Comment by ElevenWilson** [comment]
8 words might be misspelled, please check them: connait, développeuses, formatage, Gehannin, lang, login, marianne, marishka

**Comment by ElevenWilson** [comment]
10 words might be misspelled, please check them: baseurl, Eviter, Gehannin, jpg, lang, linformatique, login, marianne, marishka, png

**Comment by ElevenWilson** [comment]
10 words might be misspelled, please check them: baseurl, Eviter, Gehannin, jpg, lang, linformatique, login, marianne, marishka, png

**Comment by ElevenWilson** [comment]
14 words might be misspelled, please check them: baseurl, Edx, Gehannin, Goldwasser, jpg, lang, linformatique, login, marianne, marishka, MOOCs, png, Shafi, Udemy

**Comment by ElevenWilson** [comment]
14 words might be misspelled, please check them: baseurl, Edx, Gehannin, Goldwasser, jpg, lang, linformatique, login, marianne, marishka, MOOCs, png, Shafi, Udemy

**Comment by ElevenWilson** [comment]
14 words might be misspelled, please check them: baseurl, Edx, Gehannin, Goldwasser, jpg, lang, linformatique, login, marianne, marishka, MOOCs, png, Shafi, Udemy

**Comment by ElevenWilson** [comment]
14 words might be misspelled, please check them: baseurl, Edx, Gehannin, Goldwasser, jpg, lang, linformatique, login, marianne, marishka, MOOCs, png, Shafi, Udemy

**Comment by ElevenWilson** [comment]
15 words might be misspelled, please check them: baseurl, Edx, Gehannin, Goldwasser, jpg, lang, linformatique, l'IT, login, marianne, marishka, MOOCs, png, Shafi, Udemy

**Comment by ElevenWilson** [comment]
15 words might be misspelled, please check them: baseurl, Edx, Gehannin, Goldwasser, jpg, lang, linformatique, l'IT, login, marianne, marishka, MOOCs, png, Shafi, Udemy

**Comment by ElevenWilson** [comment]
15 words might be misspelled, please check them: baseurl, Edx, Gehannin, Goldwasser, jpg, lang, linformatique, l'IT, login, marianne, marishka, MOOCs, png, Shafi, Udemy

**Comment by ElevenWilson** [comment]
15 words might be misspelled, please check them: baseurl, Edx, Gehannin, Goldwasser, jpg, lang, linformatique, l'IT, login, marianne, marishka, MOOCs, png, Shafi, Udemy

**Review by ElevenTom — APPROVED** [review]

**Pull request merged**